test/eslint-config.js

```diff
@@ -0,0 +1,165 @@
+module.exports = {
+	parser: 'babel-eslint',
+	extends: [
+		'wordpress',
```

Comment:
Should we copy the WordPress rules here instead of using them as plugin. If I understand correctly this config is meant to be the new WordPress default eslint config.

Reply:
I assume we need to keep both for some time until merge happens. Otherwise, we will have to keep those 2 in sync. I don't have a strong opinion on this one. I think @ntwb will have some opinions here.